README.md

```diff
@@ -1,55 +1,82 @@
-
-#### iOS
-
-Add the NSLocationWhenInUsageDescription and NSLocationUsageDescription values to your info.plist file. This is the message that is displayed to your user when the permission for their location is requested. For instance,
-
-```
-<key>NSLocationWhenInUseUsageDescription</key>
-<string>TODO NSLocationWhenInUsageDescription</string>
-<key>NSLocationUsageDescription</key>
-<string>TODO NSLocationUsageDescription</string>
-```
-
-#### Android
-
-Add the location permission to your AssemblyInfo.cs file.
-
-```
-[assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
-```
+1. ### Add Package
+
+   #### .Net Mobile / WinUI
+   
+   Add the GeolocatorService.Uno.WinUI nuget package to your project heads.
+   
+   To have access to reactive extensions such as `GetAndObserveLocation`, add the `GeolocatorService.Reactive.Uno.WinUI` nuget package to your project.
+   
+   #### Xamarin / UWP
+   Add the GeolocatorService nuget package to your project heads.
+   
+   To have access to reactive extensions such as `GetAndObserveLocation`, add the `GeolocatorService.Reactive` nuget package to your project.
+
+1. ### Add the relevant permissions
+
+   #### Windows
+   
+   Add the Location capability in your manifest.
+
+   ``` xml
+    <!-- Package.appxmanifest -->
+    <Package ...>
+        <!-- (...) -->
+        <Capabilities>
+            <DeviceCapability Name="location"/>
+        </Capabilities>
+    </Package>
+   ```
+    <details>
+        <summary>Visual Studio appxmanisfest UI</summary>
+
+    ![appxmanifest](./doc/img/appxmanisfest.png)
+    </details>
+   
+   #### iOS
+   
+   Include the `NSLocationWhenInUsageDescription` and `NSLocationUsageDescription` entries in your info.plist file. These values define the message shown to users when they are asked for location permission. For instance,
+   
+   ``` xml
+    <!-- info.plist -->
+    <key>NSLocationWhenInUseUsageDescription</key>
+    <string>TODO NSLocationWhenInUsageDescription</string>
+    <key>NSLocationUsageDescription</key>
+    <string>TODO NSLocationUsageDescription</string>
+   ```
+   
+   #### Android
+   
+   Add the location permission to your AssemblyInfo.cs file.
+   
+   ``` cs
+    // AssemblyInfo.cs
+    [assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
+   ```
+
+1. ### Request permission
+
+   After you have added the relevant location permission info to your project configuration, locate the spot in your code where you want to ask the user for their location permission. This [iOS guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/requesting-permission/) or this [Android guidelines](https://developer.android.com/training/permissions/requesting) may help you decide. This must be done before accessing their location. Add the following line to do so:
+   
+   ``` cs
+   var isGranted = await locationService.RequestPermission(CancellationToken.None);
+   ```
```

Comment:
There should be another step after the permission to get the user coordinates since that's the goal of the recipe.